public/src/js/utils/observable-numeric.js

```diff
@@ -0,0 +1,16 @@
+import ko from 'knockout';
+
+export default function(initialValue) {
+    var actual = ko.observable(initialValue);
+    return ko.dependentObservable({
+        read: function ()
+        {
```

Comment:
oh, actually also in ES6 you can just do `read()` as shorthand for `read: function()`